components/vf-indicator-loader/vf-indicator-loader.scss

```diff
@@ -0,0 +1,51 @@
+// vf-indicator-loader
+
+// **Thinking about deleting this file?**
+// If your component needs no CSS/Sass, we still recommend leaving the
+// scss files in place. As this is primarily a CSS framework, it is better to
+// leave the empty files so you know a file wasn't accidently omitted.
+// If you don't have any Sass, you can trim this block down to:
+// "This page was intentionally left blank"
+
+@import 'package.variables.scss';
+// Debug information from component's `package.json`:
+// ---
+/*!
+ * Component: #{map-get($componentInfo, 'name')}
+ * Version: #{map-get($componentInfo, 'version')}
+ * Location: #{map-get($componentInfo, 'location')}
+ * Build time: #{map-get($componentInfo, 'buildTimeStamp')}
+ */
+
+@import 'vf-indicator-loader.variables.scss';
+
+// You need to add this Sass file to ./components/vf-componenet-rollup/index.scss
+
+.vf-indicator-loader--spinner__circle {
+  width: 32px;
+  height: 32px;
+  color: $vf-indicator-loader--border-color;
```

Comment:
I have some suggestion on how to use the design tokens here. Might be clearest/best if I just do that and then we can look at the diff.

Reply:
@nitinja I still need to update this Sass, but if you could please look at the other PR comment.

Reply:
I've addressed this with commit https://github.com/visual-framework/vf-core/pull/1631/commits/0c370a6d600bd834fd3f090b0a8a323f9d347170